airflow/configuration.py

```diff
@@ -343,21 +343,25 @@ def mkdir_p(path):
 
 # Setting AIRFLOW_HOME and AIRFLOW_CONFIG from environment variables, using
 # "~/airflow" and "~/airflow/airflow.cfg" respectively as defaults.
+AIRFLOW_HOME_VAR_NAME = "AIRFLOW_HOME"
+AIRFLOW_CONFIG_VAR_NAME = "AIRFLOW_CONFIG"
 
-if 'AIRFLOW_HOME' not in os.environ:
-    AIRFLOW_HOME = expand_env_var('~/airflow')
-else:
+if os.environ.get(AIRFLOW_HOME_VAR_NAME):
     AIRFLOW_HOME = expand_env_var(os.environ['AIRFLOW_HOME'])
+else:
+    AIRFLOW_HOME = expand_env_var('~/airflow')
```

Comment:
If you're going to use a default, it would be handy to log that you're using the default.